{"name": "测试输入：{\"grant_type\":\"password\",\"username\":\"\",\"password\":\"Sutpc@2020\",\"imageCode\":\"9527\"}", "status": "passed", "parameters": [{"name": "inData", "value": "'{\"grant_type\":\"password\",\"username\":\"\",\"password\":\"Sutpc@2020\",\"imageCode\":\"9527\"}'"}, {"name": "respData", "value": "'{\"error\": \"invalid_grant\",\"error_description\": \"Bad credentials\"}'"}], "start": 1619079341817, "stop": 1619079342018, "uuid": "30d8cfc0-8bd0-4835-a3ec-82784a5d943f", "historyId": "accf1ae533998cafce56a035c5e383fb", "testCaseId": "eec049e373b40a29ff5c4ae98ab4d8f6", "fullName": "test_login.TestLogin#test_login_s2", "labels": [{"name": "feature", "value": "登录模块"}, {"name": "story", "value": "登录接口"}, {"name": "severity", "value": "critical"}, {"name": "epic", "value": "TP系统"}, {"name": "tag", "value": "异常账户登陆"}, {"name": "tag", "value": "Login(order=1)"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-5R42TFK"}, {"name": "thread", "value": "12788-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login"}]}